website/docs/r/mysql_flexible_server.html.markdown

```diff
@@ -121,6 +121,8 @@ The following arguments are supported:
 
 ~> **NOTE:** The `private_dns_zone_id` is required when setting a `delegated_subnet_id`. The `azurerm_private_dns_zone` should end with suffix `.mysql.database.azure.com`.
 
+* `public_network_access` - (Optional) Whether public network access is allowed for the MySql Flexible Server. Possible values are `Enabled`, `Disabled`. Defaults to `Enabled`.
```

Comment:
Seems `Defaults to `Enabled`.` needs to be removed.

Reply:
Good catch, thanks @neil-yechenwei . I will update the docs according to the changes.

Reply:
Done.

Reply:
Technically, it does default to `Enabled` unless the server has VNet integration configured, in which case it defaults to `Disabled. I updated the documentation to explain this.